docs/sim/observations.rst

```diff
@@ -21,20 +21,22 @@ is a Python `NamedTuple` with the following fields:
     * `reached_max_episode_steps` - `True` if the vehicle has reached its max episode steps
 * `ego_vehicle_state` - a `VehicleObservation` `NamedTuple` for the ego vehicle with the following fields:
     * `id` - a string identifier for this vehicle
-    * `position` - A 3D numpy array (x, y, z) of the center of the vehicle bounding box's bottom plane
+    * `position` - A 3D array (x, y, z) of the center of the vehicle bounding box's bottom plane
     * `bounding_box` - `Dimensions` data class for the `length`, `width`, `height` of the vehicle
     * `heading` - vehicle heading in radians
     * `speed` - agent speed in m/s
-    * `steering` - angle of front wheels in radians
+    * `steering` - angle of front wheels in radians within [-1:1]
```

Comment:
Can we check whether the steering output is in radians between [-math.pi, math.pi] or is in normalised numbers within [-1, 1]?

Reply:
I seem to have made a mistake here, it should be just radians.

Reply:
I believe I had it confused with the inputs for controllers.